Package.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+    name: "Optimizely",
+    platforms: [
+        .iOS(.v9),
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(name: "Optimizely",
+                 targets: ["Optimizely"])
+    ],
+    targets: [
+        .target(name: "Optimizely", path: "Sources")
+    ],
+    swiftLanguageVersions: [.v5]
```

Comment:
Do we want to change to use Swift 5?

Reply:
Yep. Package.swift is already changed to Swift 5. I'll fix others to be consistent (Swift5+).

Reply:
I have tested with CocoaPod, Carthage, and SPM (Xcode 11). Found a couple of issues and fixed. All looks good. I'll make sure again when releasing.